build-manifest.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+      "name": "hopsworks/mysqld_exporter",
+      "version": "env:VERSION",
+      "dockerFile": ""
```

Comment:
Is this multi-platform?

Reply:
nope, it require also changing the build process with also build on mac but haven't looked into that in details since i think mysql exporter anyway at the moment is working only on Hopsworks